pygsti/tools/leakage.py

```diff
@@ -359,10 +388,46 @@ def u2x2_to_9x9_superoperator(u2x2):
 
 def lagoified_gopparams_dicts(gopparams_dicts: List[Dict]) -> List[Dict]:
     """
-    Return a modified deep-copy of gopparams_dicts, where the modifications
-    strip out gauge optimization over the TPSpam gauge group and apply a 
-    few other changes appropriate for leakage modeling.
+    goppparams_dicts is a list-of-dicts (LoDs) representation of a gauge optimization suite
+    suitable for models without leakage (e.g., a model of a 2-level system).
+
+    This function returns a new gauge optimization suite (also in the LoDs representation)
+    by applying leakage-specific modifications to a deep-copy of gopparams_dicts.
+
+    Example
+    -------
+    Suppose we have a ModelEstimateResults object called `results` that includes a
+    CPTPLND estimate, and we want to update the models of that estimate to include
+    two types of leakage-aware gauge optimization.
+
+        #
+        # Step 1: get the input to this function
+        #
+        estimate = results.estimates['CPTPLND']
+        model    = estimate.models['target']
+        stdsuite = GSTGaugeOptSuite(gaugeopt_suite_names=('stdgaugeopt',))
+        gopparams_dicts = stdsuite.to_dictionary(model)['stdgaugeopt']
+        
+        #
+        # Step 2: use this function to build our GSTGaugeOptSuite.
+        #
+        s = lagoified_gopparams_dicts(gopparams_dicts)
+        t = lagoified_gopparams_dicts(gopparams_dicts)
+        t[-1]['gates_metric'] = 'fidelity'
+        t[-1]['spam_metric']  = 'fidelity'
```

Comment:
The docstring made me think your new gaugeopt suite would be using fidelity as the default, but you're using 'frobenius squared' as far as I can tell.